.github/workflows/sonarcloud.yml

```
@@ -1,86 +1,77 @@
             -Dsonar.organization=learnathon-by-geeky-solutions
             -Dsonar.host.url=https://sonarcloud.io
-            -Dsonar.sources=${{ steps.sonar-params.outputs.sources }}
-            -Dsonar.tests=${{ steps.sonar-params.outputs.tests }}
-            -Dsonar.java.binaries=${{ steps.sonar-params.outputs.binaries }}
-            -Dsonar.java.test.binaries=${{ steps.sonar-params.outputs.test_binaries }}
-            -Dsonar.coverage.jacoco.xmlReportPaths=${{ steps.sonar-params.outputs.jacoco_reports }}
+            -Dsonar.sourceEncoding=UTF-8
+            -Dsonar.java.source=21
+            -Dsonar.coverage.jacoco.xmlReportPaths=build/reports/jacoco/aggregate/jacoco.xml
+            -Dsonar.coverage.exclusions=**/config/**,**/exception/**,**/model/**,**/dto/**,**/*Application*
+           
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)